tests/providers/test_braket_job.py

```diff
@@ -25,5 +25,6 @@ def test_job(self):
         self.assertTrue(job.result().results[0].data.counts, {"00": 1})
         self.assertTrue(job.result().results[0].shots, 100)
         self.assertTrue(job.result().results[0].status, "COMPLETED")
+        self.assertTrue(job.result().get_memory(), ["00"])
```

Comment:
I think it should be `assertEqual` as was pointed out here https://github.com/qiskit-community/qiskit-braket-provider/issues/67. 

If you can change it your PR will close 3 issues :) Amazing job!

Reply:
Sounds good. I was expected #67 to be fixed in another PR but I can do this later.